zanata-client-commands/src/main/java/org/zanata/client/commands/push/RawPushCommand.java

```diff
@@ -151,12 +323,17 @@ public void run() throws IOException {
                     .getCurrentModule());
             return;
         } else {
-            log.info("Found source documents:");
+            consoleInteractor.printfln(DisplayMode.Information,
+                "Found source documents:");
             for (String docName : localDocNames) {
                 if (docsToPush.contains(docName)) {
-                    log.info("           {}", docName);
+                    DocumentType fileType = getFileType(filteredDocTypes,
+                        getExtensionFor(docName));
+                    consoleInteractor.printfln(DisplayMode.Information,
+                        "           " + docName + " [" + fileType.name() + "]");
                 } else {
-                    log.info("(to skip)  {}", docName);
+                    consoleInteractor.printfln(DisplayMode.Information,
+                        "(to skip)  " + docName);
```

Comment:
- [ ] String cannot be localised.

From the look of the method name, `printfln(...)` probably allows placeholders, please use them.